assets/js/modules/reader-revenue-manager/components/dashboard/RRMSetupSuccessSubtleNotification.js

```diff
@@ -98,6 +99,8 @@ function RRMSetupSuccessSubtleNotification() {
 				publicationOnboardingState
 			);
 		}
+
+		dismissNotice();
```

Comment:
It looks like this was mistakenly added back from the merge conflict. This was removed as part of a recent PR.

```suggestion
```